srml/transaction-payment/src/lib.rs

```diff
@@ -227,7 +227,9 @@ impl<T: Trait + Send + Sync> SignedExtension for ChargeTransactionPayment<T>
 mod tests {
 	use super::*;
 	use codec::Encode;
-	use support::{parameter_types, impl_outer_origin, impl_outer_dispatch};
+	use support::{
+		parameter_types, impl_outer_origin, impl_outer_dispatch, traits::ExistenceRequirement
```

Comment:
```suggestion
		parameter_types, impl_outer_origin, impl_outer_dispatch, 
		traits::ExistenceRequirement,
```

(not par the guideline but IMO a very good convention: newline per sub::module)